pkg/manifestgen/sourcesecret/options.go

```diff
@@ -54,12 +60,18 @@ type Options struct {
 	Keypair             *ssh.KeyPair
 	Username            string
 	Password            string
-	CAFile              []byte
-	CertFile            []byte
-	KeyFile             []byte
+	CACrt               []byte
+	TlsCrt              []byte
+	TlsKey              []byte
```

Comment:
```suggestion
	TLSCrt              []byte
	TLSKey              []byte
```

In addition: any specific reason to use "crt" instead of "cert"? Which is prettier to the eyes, and not much longer.